<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(233, 255, 243);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Input">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Serif&quot;;</string>
    </property>
    <property name="text">
     <string>ввод данных</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Union">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Serif&quot;;</string>
    </property>
    <property name="text">
     <string>объединение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Intersection">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>пересечение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Difference">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>разность</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SymDifference">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>сим. разность</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>241</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>241</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="resultTable">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>241</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>сохранение</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>inputData()</slot>
 </slots>
</ui>
